services/director-v2/tests/unit/with_dbs/test_modules_comp_scheduler_dask_scheduler.py

```diff
@@ -80,7 +80,14 @@ def minimal_dask_scheduler_config(
     monkeypatch.setenv("DIRECTOR_V2_POSTGRES_ENABLED", "1")
     monkeypatch.setenv("COMPUTATIONAL_BACKEND_DASK_CLIENT_ENABLED", "1")
     monkeypatch.setenv("COMPUTATIONAL_BACKEND_ENABLED", "1")
-    monkeypatch.setenv("R_CLONE_S3_PROVIDER", "MINIO")
+    monkeypatch.setenv("R_CLONE_PROVIDER", "MINIO")
```

Comment:
question: you had to change a lot of test monkeypatches. Would it not be better to disable it by default in conftest and enable it in only the tests that use it?